src/ProfiledAot/build.proj

```diff
@@ -0,0 +1,19 @@
+<Project DefaultTargets="Record">
+  <Import Project="../../Directory.Build.props" />
+  <PropertyGroup>
+    <RuntimeIdentifier Condition=" '$(RuntimeIdentifier)' == '' ">android-arm64</RuntimeIdentifier>
+    <Intermediate>src/obj/$(App)/</Intermediate>
+    <CSProj>$(Intermediate)$(App).csproj</CSProj>
+    <EnvVars>DOTNET_MULTILEVEL_LOOKUP=0</EnvVars>
+  </PropertyGroup>
+  <Target Name="Record">
+    <Error Condition=" '$(App)' != 'maui' and '$(App)' != 'maui-blazor' " Text="Please pass -p:App=maui or -p:App=maui-blazor" />
+    <RemoveDir Directories="$(Intermediate)" />
+    <Exec Command="&quot;$(DotNetToolPath)&quot; new $(App) -o $(Intermediate)" EnvironmentVariables="$(EnvVars)" />
+    <!-- MainPage.xaml.cs uses CommonMethod.cs -->
+    <Copy SourceFiles="src/MainPage-$(App).cs" DestinationFiles="$(Intermediate)MainPage.xaml.cs" />
```

Comment:
This now invokes the template and copies one file over top. So a lot less code now.